frontend/app/assets/javascripts/spree/frontend/multi_currency.js

```diff
@@ -0,0 +1,11 @@
+$(() =>
+  $('#currency').on('change', function () {
+    return $.ajax({
+      type: 'GET',
+      url: $(this).data('href'),
+      data: {
+        currency: $(this).val()
+      }
+    }).done(() => window.location.reload())
```

Comment:
@PSzyszkaSpark we cannot use arrow functions with the asset pipeline, we need to stick to good old functions